core/src/test/java/com/datastax/dse/driver/internal/core/data/geometry/DefaultLineStringTest.java

```diff
@@ -94,13 +95,17 @@ public void should_load_from_well_known_binary() {
   }
 
   @Test
-  public void should_parse_valid_geo_json() {
-    assertThat(LineString.fromGeoJson(json)).isEqualTo(lineString);
+  public void should_create_and_parse_geo_json() {
+
+    String geoJsonStr = lineString.asGeoJson();
+    assertThat(JSONUtils.isValidJsonMap(geoJsonStr)).isTrue();
+    LineString newLineString = LineString.fromGeoJson(geoJsonStr);
+    assertThat(newLineString).isEqualTo(lineString);
   }
 
   @Test
-  public void should_convert_to_geo_json() {
-    assertThat(lineString.asGeoJson()).isEqualTo(json);
+  public void should_parse_legacy_geo_json() {
+    assertThat(LineString.fromGeoJson(legacyJson)).isEqualTo(lineString);
```

Comment:
I wanted to also make sure we could validate any serialized strings of the old format that might be out there somewhere.